tests/Common/MetadataTest.php

```diff
@@ -239,6 +239,86 @@ public function testTableMetadata()
         $this->assertEquals($table['bucket']['metadata'][0]['value'], $md['value']);
     }
 
+    public function testTableMetadataBulk()
+    {
+        $tableId = $this->getTestBucketId() . '.table';
```

Comment:
To sem asi ani nechtěl vidět, že na to není metoda, ale každý test si to skládá sám :crying_cat_face:

Reply:
To byl takový povzdech, ale getMetadataTestTableId v té třídě by mohlo byt

Reply:
upraveno